WordPress/openverse
# Update the global audio player

## Fixes



Related to #2910 by @fcoveram

## Description


This PR updates the Global audio player to match the design requirements in #2910.

<img width="731" alt="Screenshot showing the previous version of the global audio component compared to the updated version" src="https://github.com/WordPress/openverse/assets/15233243/02f20097-a28a-4ec1-801f-4985f4615ec8">


It also fixes the problem we had with global audio player: it sometimes appeared on the home page, but didn't appear on the single result page. This was caused by the updates in the layouts, when I accidentally added the component into the wrong layout.
The old no longer used `VCloseButton` component is also removed.

## Testing Instructions

Open the app on a mobile-width screen and search for something. When you click on Play button, you should be able to see the global player. You can seek, pause or play using it. If you click Play on a different audio, the audio should be updated. If you click on the audio title, the single result should open, and the global player should stay on the screen with the same audio playing.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
